recipes/zeus_expected/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.cmake import cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class ZeusExpectedConan(ConanFile):
+    name = "zeus_expected"
+
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zeus-cpp/expected"
+    description = "Backporting std::expected to C++17."
+    topics = ("cpp17", "expected")
+    license = "MIT"
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "gcc": "7",
+            "clang": "5",
+            "Visual Studio": "15.7",
+            "apple-clang": "10",
+        }
+
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if min_version and Version(self.settings.compiler.version) < min_version:
+            raise ConanInvalidConfiguration(f"{self.name} requires C++{self._min_cppstd}, which your compiler does not support.")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(
+            self,
+            **self.conan_data["sources"][self.version],
+            destination=self.source_folder,
+            strip_root=True
+        )
+
+    def package(self):
+        copy(
+            self,
+            "LICENSE",
+            src=self.source_folder,
+            dst=os.path.join(self.package_folder, "licenses"),
+        )
+        copy(
+            self,
+            "*",
+            src=os.path.join(self.source_folder, "include"),
+            dst=os.path.join(self.package_folder, "include"),
+        )
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "zeus_expected")
+        self.cpp_info.set_property("cmake_target_name", "zeus::expected")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "zeus::expected")
        if is_msvc(self):
            self.cpp_info.cxxflags.append("/Zc:__cplusplus")
```

Reply:
Resolved by a [new upstream version](https://github.com/zeus-cpp/expected/releases/tag/v1.1.0) that don't need `"/Zc:__cplusplus` anymore.